### PR TITLE
Epobutt bugfix

### LINKED ISSUE #6618: tight_layout vertically compresses PSD plots
The extension of spatial colors from Raw to Epochs in #6451 has revealed a bug with the use of tight_layout.

https://github.com/mne-tools/mne-python/blob/cc836c2635b7a720ec6c60074a1997fe52d7ca2f/mne/viz/utils.py#L3064-L3066

Tight layout often (but not always!) vertically compresses the PSD plots when spatial_colors are used with Epochs. I thought at first this had something to do with the changes in #6451, but it turns out to have always been there. It just wasn't apparent before because it only happens when the channels are relatively flat and uniform - which can sometimes happen with the multitaper psd method used by method Epochs.plot_psd, and also because because tight_layout was never used in the average=True case, which up until #6451 was the only option available for Epochs.plot_psd.

You can test that this is the case by taking the latest stable release and producing a vertically compressed PSD plot even for Raw.plot_psd, but setting the n_fft variable to an extremely low value - thus simulating the very broad window used in Epochs with multitaper fft, and the resultant flat, uniform PSDs.

What would then be the sensible thing to do here? Is simply removing the call to tight_layout an option? It was after all never used for average=True, and that seems fine?

### PR DESCRIPTION
**Pull request by jshanna100**
Closes #6618

The first figure in examples/plot_compute_raw_data_spectrum now uses average=False.

### CODE REVIEW

**Comment by jshanna100** [comment]
Yeah, looks good on my stuff, too.

File: doc/whats_new.rst [hunk 1]
```diff
@@ -61,6 +61,8 @@ Changelog
 Bug
 ~~~
 
+- Fix horizontal spacing issues in :func:`mne.viz.utils._plot_psd` by `Jeff Hanna`
```
Comment (agramfort) [inline doc/whats_new.rst]:
```suggestion
- Fix horizontal spacing issues in :meth:`mne.io.Raw.plot_psd` by `Jeff Hanna`
```

```

don't document the private function but the function in the public API.

**Comment by massich** [comment]
I pushed minor changes. Merge when green.

**Comment by massich** [comment]
Thx @jshanna100

**Pull request merged**